README.md

```diff
@@ -32,6 +32,14 @@ If you've globally installed `stylelint-config-recommended` using the `-g` flag,
 }
 ```
 
+Since [stylelint 9.7.0](https://github.com/stylelint/stylelint/blob/9.7.0/CHANGELOG.md#970), you can simply use the globally installed configuration name instead of the absolute path:
+
+```json
+{
+  "extends": "stylelint-config-recommended"
+}
+```
+
```

Comment:
Can you please update the existing section instead of adding a new section please, lines 27-33

Reply:
Right, thanks, this is fine as is then, sorry about that